package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "jest-puppeteer-react",
-    "version": "3.0.3",
+    "version": "3.0.4",
```

Comment:
Don’t update the package version, this will to be done on release

Reply:
It's already released, but can't push to master